packages/core/src/components/private/BaseInputRange/index.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
```

Comment:
This can probably just be in `Range.tsx` since it's not used anywhere else.

Reply:
moved to `Range/` as `BaseRange` so we can keep it separate from the `Form`-wrapped one (`Range/index`)